[2025-02-21T17:28:01.380+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kafka_spark_dag.kafka_data_stream manual__2025-02-21T17:27:47.921344+00:00 [queued]>
[2025-02-21T17:28:01.544+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kafka_spark_dag.kafka_data_stream manual__2025-02-21T17:27:47.921344+00:00 [queued]>
[2025-02-21T17:28:01.548+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-02-21T17:28:01.663+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): kafka_data_stream> on 2025-02-21 17:27:47.921344+00:00
[2025-02-21T17:28:01.772+0000] {standard_task_runner.py:57} INFO - Started process 3539 to run task
[2025-02-21T17:28:01.820+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'kafka_spark_dag', 'kafka_data_stream', 'manual__2025-02-21T17:27:47.921344+00:00', '--job-id', '75', '--raw', '--subdir', 'DAGS_FOLDER/dag_kafka_spark.py', '--cfg-path', '/tmp/tmpw2qmg75e']
[2025-02-21T17:28:01.840+0000] {standard_task_runner.py:85} INFO - Job 75: Subtask kafka_data_stream
[2025-02-21T17:28:02.850+0000] {task_command.py:416} INFO - Running <TaskInstance: kafka_spark_dag.kafka_data_stream manual__2025-02-21T17:27:47.921344+00:00 [running]> on host 50e7bf26003b
[2025-02-21T17:28:04.127+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='kafka_spark_dag' AIRFLOW_CTX_TASK_ID='kafka_data_stream' AIRFLOW_CTX_EXECUTION_DATE='2025-02-21T17:27:47.921344+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-21T17:27:47.921344+00:00'
[2025-02-21T17:28:43.443+0000] {logging_mixin.py:154} INFO - Dataset downloaded successfully.
[2025-02-21T17:28:43.696+0000] {logging_mixin.py:154} INFO - Dataset extracted successfully.
[2025-02-21T17:28:43.697+0000] {logging_mixin.py:154} INFO - No CSV file found in extracted data.
[2025-02-21T17:28:43.701+0000] {logging_mixin.py:154} INFO - Dataset preparation failed.
[2025-02-21T17:28:43.702+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-02-21T17:28:43.795+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=kafka_spark_dag, task_id=kafka_data_stream, execution_date=20250221T172747, start_date=20250221T172801, end_date=20250221T172843
[2025-02-21T17:28:43.996+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-02-21T17:28:44.554+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
